tests/modules/rigidbody/test_rigidbody.py

```diff
@@ -0,0 +1,98 @@
+"""Test rigidbody module."""
+import json
+from pathlib import Path
+
+from haddock.modules.sampling.rigidbody import HaddockModule as RigidBody
+
+
+RECIPE_PATH = Path(
+    Path(__file__).resolve().parents[3],
+    'src/haddock/modules/sampling/rigidbody'
+    )
+DEFAULT_CONFIG = Path(RECIPE_PATH, "defaults.cfg")
+DATA_PATH = Path(Path(__file__).resolve().parent, 'data')
+
+CONF_PARAMS = {
+    'config_path': DATA_PATH,
+    'mode': 'local'
+    }
+
+
+def initialize():
```

Comment:
Why don't you create an initialized `json` directly instead of having  a function to create it? what are you testing? I would suggest creating the function and loading the json in the test.

Reply:
Because its my first time with pytest and I don't know how its best to do things 😄

Reply:
pytests just runs functions and see if they raise any exception. normally with pytest, you receive some parameters (optionally) and assert a result against an expectation. Or, you run something and see if some errors come out. Go for it :+1: